tests/factories/StateFactory.php

```diff
@@ -22,8 +22,8 @@ protected function createDefault(): FactoryInterface
     {
         return parent::createDefault()
             ->withActive(true)
-            ->withIdCountry(1)
-            ->withIdZone(1);
+            ->withIdCountry(Country::getByIso('NL'))
+            ->withIdZone(Zone::first()->id);
```

Comment:
Hoe weet je dat de eerste zone de zone voor Nederland is?

Reply:
maakt niet uit, is gewoon test data